addons/website_sale/tests/test_website_sale_cart_abandoned.py

```diff
@@ -129,3 +129,33 @@ def test_search_abandoned_cart(self):
         self.assertTrue(self.so1after.id in non_abandoned)
         self.assertTrue(self.so2after.id in non_abandoned)
         self.assertFalse(self.so2before_but_public.id in abandoned)
+
+    def test_send_abandoned_cart_email(self):
```

Comment:
The test is fine as is, but I would suggest patching `send_mail` and catching the calls to it (possibly verifying values) to avoid rendering a whole mail, queries etc as it's not _really_ necessary for this test.